bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/DiscoveryResource.java

```diff
@@ -91,7 +101,9 @@ public void onFinished() {
             }
         });
 
-        return Response.ok(discoveryServiceRegistry.getMaxScanTimeout(bindingId)).build();
+        DiscoveryServiceDTO dto = new DiscoveryServiceDTO();
+        dto.timeout = discoveryServiceRegistry.getMaxScanTimeout(bindingId);
```

Comment:
Yeah, but maybe we will never add anything and then it would be all overhead work in vain...
And for this use case, I really don't see anything at the horizon... It is a bit like changing method signatures to always return List<String> instead of String, just to be future proof ;-)